src/mainwindow.ui

```diff
@@ -117,10 +117,17 @@
     <addaction name="separator"/>
     <addaction name="action_anim_delete"/>
    </widget>
+   <widget class="QMenu" name="menuUtilities">
+    <property name="title">
+     <string>Utilities</string>
```

Comment:
Why do you think this (`Copy selection to frame`) belongs to a new `Utilities` menu instead of putting it into the `Animation` menu to the other frame related commands?